src/rendering.ts

```diff
@@ -905,16 +917,23 @@ function renderBar(
     // console.log(dataset);
     // console.log(barInfo);
     // console.log("%d/%d", currBarSet, totalNumOfBarSets);
-
+    
```

Comment:
Just a minor nitpick and I'll fix this after the merge, but please try to avoid adding whitespace like this

Reply:
Thanks for catching that!